### DISCUSSION TITLE
Xomega and firebird databases

### DISCUSSION THREAD

**Discussion by jimmaguire** [Xomega-Net/Xomega.Net4VS#20] [Q&A]
Does Xomega support firebird databases, say via ado.net?

**Comment by XomegaNet** [comment]
Xomega uses Entity Framework Core, so the apps it can build can support any database that EF Core supports, which I believe includes the Firebird. However, Xomega solution template currently supports only SQL Server and PostgreSQL databases, and those are the DBs that you can import your model from.

You may need to map the base logical types in the model to Firebird data types to configure the generation of EF entities for Firebird.

Is full support for Firebird databases something you'd like to see added to Xomega?

**Comment by jimmaguire** [comment]
Yes, very interested in your product but I need to redo my app's client side from Telerik JQuery Hybrid that has been discontinued.  So I don't think I can wait.
